ssl/statem/statem_srvr.c

```diff
@@ -3000,131 +3000,82 @@ static int tls_process_cke_rsa(SSL *s, PACKET *pkt)
      * (SSL_MAX_MASTER_KEY_LENGTH). Reject overly short RSA keys because
      * their ciphertext cannot accommodate a premaster secret anyway.
      */
-    if (RSA_size(rsa) < SSL_MAX_MASTER_KEY_LENGTH) {
+    if (EVP_PKEY_size(rsa) < SSL_MAX_MASTER_KEY_LENGTH) {
```

Comment:
That should probably say RSA_PKCS1_PADDING_SIZE + SSL_MAX_MASTER_KEY_LENGTH

Reply:
Changed.